sql/updates/world/master/xxxx_xx_xx_x1_world.sql

```diff
@@ -0,0 +1,97 @@
+UPDATE `trinity_string` SET `content_default`='Up to %i expansions are allowed now.' WHERE  `entry`=61;
```

Comment:
The PRs depend on each other though so making separate PRs now would make the other have conflicts with master so not sure that is possible in a good way.

Guess I need to merge them then if there is no other way.